db/operations.go

```diff
@@ -82,25 +86,26 @@ func (o *Operation) query(typeGetter TypeGetter) (string, error) {
 	if o.opType == OperationTypeDelete {
 		return fmt.Sprintf("DELETE FROM %s.%s WHERE %s = %s", o.schemaName, o.tableName, o.primaryKeyColumnName, o.primaryKey), nil
 	}
-	keys, values, err := prepareColValues(o.tableName, o.data, typeGetter)
+
+	columns, values, err := prepareColValues(o.tableName, o.data, typeGetter)
 	if err != nil {
 		return "", fmt.Errorf("preparing column-values: %w", err)
 	}
 	if o.opType == OperationTypeInsert {
 		return fmt.Sprintf("INSERT INTO %s.%s (%s) VALUES (%s)",
 			o.schemaName,
 			o.tableName,
-			strings.Join(keys, ","),
+			strings.Join(columns, ","),
 			strings.Join(values, ","),
 		), nil
 	}
 
 	var updates []string
-	for i := 0; i < len(keys); i++ {
+	for i := 0; i < len(columns); i++ {
 		// FIXME: merely using %s for key can lead to SQL injection. I
```

Comment:
This comment does not apply right? We correctly escape wrong characters in the statement?